Scripts/whitesource-spdx.yml

```diff
@@ -45,6 +45,32 @@ jobs:
             echo WS_URL: $WS_URL
             echo === Installing WhiteSource SBOM Generator ===
             pip  install ws_sbom_generator
+            echo === Checking Project State before generating SBOM ===
+            IFS="|"
+            scan_status=true
+            pass_status=("UPDATE"${IFS}"FINISH"${IFS}"DIFF")
+            fail_status=("UNKONWN"${IFS}"FAIL")
```

Comment:
UNKNOWN is spelled incorrectly....is this a bug in WSS or a typo?

Reply:
Typo